index.js

```diff
@@ -12,11 +12,12 @@ class ScrollTrigger {
     this.eventHandler = tinybounce(this.onScroll.bind(this), 10, true);
     this.dCalcBounds = tinybounce(this.calcBounds.bind(this), 10);
     this.paused = false;
+    this.disabled = false;
 
     this.calcBounds();
 
-    on(window, 'scroll', this.eventHandler);
-    on(window, 'resize', this.dCalcBounds);
+    window.addEventListener('scroll', this.eventHandler);
```

Comment:
why did this change?

Reply:
So we can unbind later. `on` and `off` have not been fixed yet on Domassist.